qa/automation/ScoringPortal.Tests.Scenarios/3. Sign In/SignIn.feature

```diff
@@ -0,0 +1,247 @@
+Feature: 3.1.1.1. Sigh in with google account. 
+
+Background:
+    Given the web page "https://qa.ddso-spot.quantori.com/" is open in the browser
+    And user has a google account with valid credentials: email {email@google.com}, password {Password!23}, assigned phone number {+11234561234}
+
+Scenario Outline: 3.1.1.1_01 Sign in with full email address if the user is not logged in
+    Given user clicked on "Sign in with Google" button
+    And webpage with a textfield is opened
+    When User enters full email address into textfield 
+    And User clicks 'Next' button
+    And webpage with textfield for password is opened
+    And user enters the valid password 
+    And user clicks 'Next' button
+    Then user is signed in
```

Comment:
I'm not sure if this line is necessary - right after pushing the button all we can see is that the main page opens.

Reply:
I agree. But i prefer to leave it as it is. I am thinking we need to ensure that the user logged in sufficiently and the application page is active